azure-pipelines.yml

```diff
@@ -26,6 +33,21 @@ jobs:
   steps:
   - template: ci/publish-steps.yml
 
+# Courtesy push on Windows
+- job: courtesy_push_windows
+  displayName: Courtesy Push
+  dependsOn: 
+  - publish_windows
+  condition: and(succeeded(), in(variables['build.reason'], 'Schedule', 'Manual'), eq(variables['courtesyPush'], 'true'))
```

Comment:
Do we need to make sure this is only on `master` or `releases\*`?

Reply:
Yeah, I overrode the triggers in the pipeline itself (that way we don't need to have 2 yaml files or complicated logic to differentiate